CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Ewon Ignition 8 Connector Changelog
 
+## v1.2.0
+
+### Major Changes
+
+* Updated to Ignition 8.1.1 SDK
+* Fixed an issue that caused the AllRealtime tag value to be lost
+* Fixed an issue that caused tag deletion to become disabled  
+* Updated the DataMailbox and M2Web APIs to use HTTP POST requests
+* Updated the DataMailbox API to use Talk2M token authentication

```

Comment:
You might want to consider rewording these. You didnt update the api, you just changed your usage of the api.

Reply:
changed